mmv1/products/apigateway/terraform.yaml

```diff
@@ -58,33 +60,38 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         min_version: beta
         name: "apigateway_api_config_basic"
         primary_resource_id: "api_cfg"
+        primary_resource_name: "fmt.Sprintf(\"tf-test-api-%s\", context[\"random_suffix\"]), fmt.Sprintf(\"tf-test-config-%s\", context[\"random_suffix\"])" # Need to pass 2 ids into a Sprintf - parent resource id also needed to identify primary resource
```

Comment:
Not sure if there's a better way to achieve this?

Reply:
not that I know of offhand.